sofy_sofy/README.md

```diff
@@ -38,15 +38,15 @@ Need help? Contact [Sofy support][5].
 
 ## Further Reading
 Additional helpful documentation, links, and articles:
-* [Blog Post][6]
-* [Documentation][7]
+* [Monitor your mobile tests with Sofy’s offering in the Datadog Marketplace][6]
+* [Sofy Documentation][7]
 
 
 [1]: https://app.datadoghq.com/integrations
 [2]: https://portal.sofy.ai
 [3]: https://portal.sofy.ai/app/user-settings?selectedTab=integration
 [4]: https://app.datadoghq.com/organization-settings/api-keys?filter=Sofy
 [5]: https://support.sofy.ai/support/tickets/new
-[6]: https://sofy.ai/blog/
+[6]: [https://sofy.ai/blog/](https://www.datadoghq.com/blog/sofy-mobile-tests/)
```

Comment:
Just confirming that the way the link is formatted is intentional as I haven't seen a link formatted like this before.

Reply:
Oops! Thanks for flagging.

```suggestion
[6]: https://www.datadoghq.com/blog/sofy-mobile-tests/
```